### PR TITLE
HDDS-2906. Added Unit Test Cases ofr CRLCodec

### PR DESCRIPTION
**Pull request by abhishekaypurohit**
## What changes were proposed in this pull request?

Added more Unit Tests for CRLCodec

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2906

## How was this patch tested?

mvn build and unit tests. 

@anuengineer

### CODE REVIEW

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/utils/CRLCodec.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ public CRLCodec(SecurityConfig securityConfig) {
    * Returns a X509 CRL from the CRL Holder.
    *
    * @param holder - Holder
-   * @return X509Certificate.
+   * @return X509CRL - X509 CRL.
    * @throws CRLException - on Error.
    */
   public static X509CRL get509CRL(X509CRLHolder holder)
```
Comment (xiaoyuyao) [inline hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/utils/CRLCodec.java]: NIT: should we rename the method from get509CRL to getX509CRL?

File: hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/security/x509/certificate/utils/TestCRLCodec.java [hunk 2]
```diff
@@ -101,11 +122,124 @@ public void testWriteCRL() throws IOException, OperatorCreationException {
...
+
+    X500Name issuer = x509CertificateHolder.getIssuer();
+    Date now = new Date();
+    X509v2CRLBuilder builder = new X509v2CRLBuilder(issuer, now);
+    builder.addCRLEntry(x509CertificateHolder.getSerialNumber(), now,
+                        CRLReason.cACompromise);
+
+    InputStream inStream = null;
+    byte[] crlBytes = TMP_CRL_ENTRY.getBytes();
+    try {
```
Comment (xiaoyuyao) [inline hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/security/x509/certificate/utils/TestCRLCodec.java]: NIT: the InputStream can be wrapped with try-with-resource.

Comment (abhishekaypurohit) [reply hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/security/x509/certificate/utils/TestCRLCodec.java]: Done.

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/utils/CRLCodec.java (hunk 1 above)
Comment (abhishekaypurohit) [reply hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/utils/CRLCodec.java]: Good point. Done.

**Comment by xiaoyuyao** [comment]
+1 I will merge it shortly.

**Pull request merged**